test/common/verifier.go

```diff
@@ -170,7 +170,7 @@ func (v *Verifier) verifyThatLogsAreNotForwardedToEchoServer(ctx context.Context
 	EventuallyWithOffset(2, func(g Gomega) {
 		logLines, err := v.getLogs(ctx, timeBeforeLogGeneration)
 		g.Expect(err).NotTo(HaveOccurred())
-		g.Expect(logLines).To(BeEmpty())
+		g.Expect(logLines).To(Or(BeEmpty(), ConsistOf(ContainSubstring("-- Mark --"))))
```

Comment:
Optionally, we can optionally add a comment describing the `-- Mark --` logs as I didn't know what is the reason behind it:

```suggestion
		// -- Mark -- is a period log emitted by rsyslogd denoting that there are no incoming logs.
		g.Expect(logLines).To(Or(BeEmpty(), ConsistOf(ContainSubstring("-- Mark --"))))
```

Reply:
The more correct description would be that it is a "heartbeat" log message which lets you know that it is working. I'll add it for clarity.